pandas/io/formats/style.py

```diff
@@ -1222,6 +1222,78 @@ class MyStyler(cls):
 
         return MyStyler
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Apply ``func(self, *args, **kwargs)``, and return the result.
+
+        .. versionadded:: 0.24.0
+
+        Parameters
+        ----------
+        func : function
+            Function to apply to the Styler.
+            ``*args``, and ``**kwargs`` are passed into ``func``.
+            Alternatively a ``(callable, data_keyword)`` tuple where
+            ``data_keyword`` is a string indicating the keyword of
+            ``callable`` that expects the Styler.
+        *args : iterable, optional
+            Positional arguments passed into ``func``.
+        **kwargs : dict, optional
+            Dictionary of keyword arguments passed into ``func``.
+
+        Returns
+        -------
+        result : object
+            The value returned by ``func``.
```

Comment:
Is this true? My understanding is that the result of `func` is used in the cells of the styler, and that a new styler is returned.

Reply:
I think you're confusing this method with some of the other Styler methods, like [Styler.applymap](https://pandas.pydata.org/pandas-docs/version/0.23.4/generated/pandas.io.formats.style.Styler.applymap.html).